### PR TITLE
document the actual interface we expect people to use

### PR DESCRIPTION
**Pull request by dulitz**
@austinmroczek is what I wrote correct? If not, let me know and I will change it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,110 @@ If you want to easily put the info into a file for sharing:
...
+
+## Recent Interface Changes
+
+* The arming control methods in TotalConnectClient have been deprecated; instead use the
+similar methods on the values of self.locations.
+
+## Likely Future Interface Changes
+
+* Previously if the usercodes dictionary was invalid, the DEFAULT_USERCODE
+was silently used. In a future release, we will crash on an invalid dictionary.
```
Comment (austinmroczek) [inline README.md:132]: Maybe "exit" instead of "crash"

File: README.md [hunk 2]
```diff
@@ -26,8 +26,110 @@ If you want to easily put the info into a file for sharing:
...
+* Previously most methods returned True on success and False on failure, with no exceptions expected. In a future release, those methods will return no value and raise subclasses of TotalConnectError on failure.
+
+If there's something about the interface you don't understand, check out the (Home Assistant integration)[https://github.com/home-assistant/core/blob/dev/homeassistant/components/totalconnect/] that uses this package, or submit an issue here.
```
Comment (austinmroczek) [inline README.md:135]: Please add another paragraph along the lines of:

During development, if you discover new status codes or other information not handled, please submit an issue to let us know, or even better submit a pull request.

**Review by austinmroczek — COMMENTED** [review]
Few minor fixes recommended

**Comment by dulitz** [comment]
PR updated with your suggestions.

**Comment by austinmroczek** [comment]
Think I forgot to tell you, with the third PR you are officially a maintainer...

**Comment by dulitz** [comment]
lolwut? :)

**Pull request merged**